### PR TITLE
Trend analysis - Either-or instead of must 

### PR DESCRIPTION
**Pull request by itachi3**

### CODE REVIEW

File: foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/actions/TrendAction.java [hunk 1]
```diff
@@ -105,11 +105,9 @@ public ActionResponse execute(TrendRequest parameter) throws QueryStoreException
...
         }
         if (null != parameter.getValues()) {
-            List<Filter> filters = Lists.newArrayList();
-            for (String value : parameter.getValues()) {
-                filters.add(new EqualsFilter(field, value));
-            }
-            parameter.getFilters().addAll(filters);
+            List<Object> values = (List) parameter.getValues();
+            Filter filter = new InFilter(field, values);
+            parameter.getFilters().add(filter);
```
Comment (r0goyal) [inline foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/actions/TrendAction.java]: @itachi3 Add test case for this.

**Comment by r0goyal** [comment]
@itachi3 Tests are failing. Check please

File: foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/actions/TrendAction.java (hunk 1 above)
Comment (itachi3) [inline foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/actions/TrendAction.java]: Will check and correct the tests cases today.

**Comment by itachi3** [comment]
We should introduce ListUtils and StringUtils to handle the checks generically. Well we can take it up when we revisit the code.

**Pull request merged**